site/_posts/2020-09-21-troubleshooting-containerized-oidc.md

```diff
@@ -255,6 +256,55 @@ The following example shows how to check for the user's request headers from the
   ... "message":"MIQ(Authenticator::Httpd#log_auth_debug)   X-REMOTE-USER-GROUPS     = \"EvmGroup-super_administrator\""}
   ```
 
+## Adjusting Log Level
+---------------------------------------------------------------------
+
+To gain additional insight to authentication issues adjust the log level to **debug** of the httpd pod,
+which can be done with the following commands:
+
+  + Login
+
+  ```bash
+  $ oc login -u admin -p <pw> https://<my-container-host>/
+  ```
+
+  + Edit the httpd configmap:
+
+  ```bash
+  $ oc edit configmaps/httpd-configs -n joev-miq
```

Comment:
you probably want to be consistent with namespace with below, maybe replace with my-miq-project